### PR TITLE
Rename PieceArray to File

### PR DESCRIPTION
**Pull request by svalasovich**

### CODE REVIEW

File: packages/ddc-client/src/DdcClient.interface.ts [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
     bucketList(offset: bigint, limit: bigint, filterOwnerId?: string): Promise<BucketStatusList>
 
-    store(bucketId: bigint, pieceArray: PieceArray, options: StoreOptions): Promise<PieceUri>
+    store(bucketId: bigint, piece: Piece, options?: StoreOptions): Promise<PieceUri>
 
-    read(pieceUri: PieceUri, options: ReadOptions): Promise<PieceArray>
+    store(bucketId: bigint, file: File, options?: StoreOptions): Promise<PieceUri>
 
-    search(query: Query): Promise<Array<PieceArray>>
+    read(url: string | URL, options: ReadOptions): Promise<File | Piece>
```
Comment (upalinski) [inline packages/ddc-client/src/DdcClient.interface.ts:23]: can we use only ddcUri?

**Review by upalinski — APPROVED** [review]

File: packages/ddc-client/src/DdcClient.interface.ts [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
 
     bucketGet(bucketId: bigint): Promise<BucketStatus>
 
     bucketList(offset: bigint, limit: bigint, filterOwnerId?: string): Promise<BucketStatusList>
 
-    store(bucketId: bigint, pieceArray: PieceArray, options: StoreOptions): Promise<PieceUri>
+    store(bucketId: bigint, piece: Piece, options?: StoreOptions): Promise<PieceUri>
 
-    read(pieceUri: PieceUri, options: ReadOptions): Promise<PieceArray>
+    store(bucketId: bigint, file: File, options?: StoreOptions): Promise<PieceUri>
```
Comment (upalinski) [inline packages/ddc-client/src/DdcClient.interface.ts:21]: store(bucketId: bigint, file: File, options?: StoreOptions): Promise<DdcUri>

File: packages/ddc-client/src/DdcClient.interface.ts [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
 
-    search(query: Query): Promise<Array<PieceArray>>
+    read(url: string | URL, options: ReadOptions): Promise<File | Piece>
```
[the remark this answers is not in the document]
Comment (upalinski) [reply packages/ddc-client/src/DdcClient.interface.ts:23]: DdcUri(bucketId: bigint, cid: string, protocol: Protocol?)

fromString(uri: String): DdcUri
// Example: https://node-0.cdn.cere.network
fromUrl(url: String): DdcUri

[the remark this answers is not in the document]
Comment (upalinski) [reply packages/ddc-client/src/DdcClient.interface.ts:23]: DdcUri.parseUrl("https://node-0.cdn.stg.cere.network")

File: packages/ddc-client/src/DdcClient.interface.ts (hunk 1 above)
Comment (svalasovich) [reply packages/ddc-client/src/DdcClient.interface.ts]: Updated

File: packages/ddc-client/src/DdcClient.interface.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (svalasovich) [reply packages/ddc-client/src/DdcClient.interface.ts]: updated

File: packages/ddc-client/src/DdcClient.interface.ts (hunk 2 above)
Comment (svalasovich) [reply packages/ddc-client/src/DdcClient.interface.ts]: updated

**Review by upalinski — APPROVED** [review]

**Pull request merged**